app/src/main/res/menu/address_book_entry_details_menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/address_book_entry_details_menu_share"
+        android:icon="@drawable/ic_share_24dp"
+        android:title="@string/share_address"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/address_book_entry_details_menu_delete"
+        android:icon="@drawable/ic_delete_24dp"
```

Comment:
Does this work on older platforms? https://stackoverflow.com/questions/35813199/how-can-i-use-vectordrawable-with-the-android-toolbar

Reply:
It works ;)